Traceback (most recent call last):
  File "notebook.py", line 171, in <module>
    schedule.run_pending()
  File "/opt/anaconda3/lib/python3.8/site-packages/schedule/__init__.py", line 822, in run_pending
    default_scheduler.run_pending()
  File "/opt/anaconda3/lib/python3.8/site-packages/schedule/__init__.py", line 100, in run_pending
    self._run_job(job)
  File "/opt/anaconda3/lib/python3.8/site-packages/schedule/__init__.py", line 172, in _run_job
    ret = job.run()
  File "/opt/anaconda3/lib/python3.8/site-packages/schedule/__init__.py", line 693, in run
    ret = self.job_func()
  File "notebook.py", line 149, in get_data_and_post
    estaciones_post = newapi.create_tweet(text=
  File "/opt/anaconda3/lib/python3.8/site-packages/tweepy/client.py", line 835, in create_tweet
    return self._make_request(
  File "/opt/anaconda3/lib/python3.8/site-packages/tweepy/client.py", line 129, in _make_request
    response = self.request(method, route, params=request_params,
  File "/opt/anaconda3/lib/python3.8/site-packages/tweepy/client.py", line 115, in request
    raise TooManyRequests(response)
tweepy.errors.TooManyRequests: 429 Too Many Requests
Too Many Requests
